src/config/redis-info.ts

```diff
@@ -6,7 +6,14 @@ export const getRedisInfo = (connectionName: string): IORedis.RedisOptions => ({
 	port: Number(envVars.REDISX_CACHE_PORT) || 6379,
 	host: envVars.REDISX_CACHE_HOST || "127.0.0.1",
 	db: 0,
+	reconnectOnError(err) {
+		const targetError = "READONLY";
 
+		if (err.message.includes(targetError)) {
+			return 1;
```

Comment:
curious why 1 and not true?

Reply:
if true is acceptible, could be `return err.messages.includes(targetError)`

Reply:
`ioredis` library expects a number, its milliseconds until reconnect, and it throws a tantrum if you pass true.